stubs/pycocotools/pycocotools/mask.pyi

```diff
@@ -1,26 +1,22 @@
-from _typeshed import Incomplete
 from typing import Any, overload
 from typing_extensions import TypeAlias
 
-from . import _EncodedRLE
+import numpy as np
+import numpy.typing as npt
 
-# TODO: Use numpy types when #5768 is resolved.
-# import numpy as np
-# import numpy.typing as npt
+from . import _EncodedRLE
 
-_NPUInt32: TypeAlias = Incomplete  # np.uint32
-_NDArrayUInt8: TypeAlias = Incomplete  # npt.NDArray[np.uint8]
-_NDArrayUInt32: TypeAlias = Incomplete  # npt.NDArray[np.uint32]
-_NDArrayFloat64: TypeAlias = Incomplete  # npt.NDArray[np.float64]
+_NPUInt32: TypeAlias = np.uint32
+_NDArrayUInt8: TypeAlias = npt.NDArray[np.uint8]
+_NDArrayUInt32: TypeAlias = npt.NDArray[np.uint32]
+_NDArrayFloat64: TypeAlias = npt.NDArray[np.float64]
 
 def iou(
     dt: _NDArrayUInt32 | list[float] | list[_EncodedRLE],
     gt: _NDArrayUInt32 | list[float] | list[_EncodedRLE],
     pyiscrowd: list[int] | _NDArrayUInt8,
 ) -> list[Any] | _NDArrayFloat64: ...
 def merge(rleObjs: list[_EncodedRLE], intersect: int = ...) -> _EncodedRLE: ...
-
-# ignore an "overlapping overloads" error due to _NDArrayInt32 being an alias for `Incomplete` for now
 @overload
 def frPyObjects(pyobj: _NDArrayUInt32 | list[list[int]] | list[_EncodedRLE], h: int, w: int) -> list[_EncodedRLE]: ...  # type: ignore[misc]
```

Comment:
We need to actually remove the outdated `type: ignore` if we're deleting the comment about why the `type ignore` used to be needed 😄

```suggestion
def frPyObjects(pyobj: _NDArrayUInt32 | list[list[int]] | list[_EncodedRLE], h: int, w: int) -> list[_EncodedRLE]: ...
```

Reply:
It becomes a different overload error, see: https://github.com/Avasam/typeshed/actions/runs/3717387421/jobs/6304685925#step:6:114

Reply:
Ah, in which case maybe we should keep the comment, but change it to "mypy complains the overloads overlap, but it still works" or something

Reply:
Sidenote, since pyright 1.1.283 it's possible to split `type: ignore` from `pyright: ignore` (and have them both work on the same line). So we could watch for obsolete type: ignore comments. I have an upcoming PR for that.

Reply:
(Though remember that mypy's --warn-unused-ignores function doesn't work properly inside typeshed because of the unique special casing mypy has for "typeshed directories" ☹️)